src/pages/getting-started/app-guides/helpdesk-widgets/index.mdx

```diff
@@ -0,0 +1,47 @@
+---
+weight: 80
+category: "getting-started"
+subcategory: "app-guides"
+title: "HelpDesk Widgets"
+tagline: "HelpDesk Widgets are web apps loaded inside the HelpDesk Application."
+desc: "HelpDesk Widgets are web apps loaded inside the HelpDesk Application. You can use them to embed and display static content, embed your SaaS service, or query external services."
+---
+
+## Introduction
+
+App Widgets are web apps loaded inside the HelpDesk Application. They live in iframes and can interact and communicate with the HelpDesk App. You can use them to:
+
+- embed and display static content, for example, knowledge base articles, conversation prompts, or context information.
+- embed your SaaS service or web app as part of the agents' workspace.
+- query external services with the ticket requester.
+- create a store with in-app purchases to sell additional services or features of your app.
+
+## Building block configuration
+
+There are three fields visible while Creating a HelpDesk Widget: **Widget source URL**, **Widget placement**, and **Opening state**.
+
+**Widget source URL** is the address of your deployed application. You can also enter `https://localhost/` while you develop the app. If you don't have the code yet but want to play around, feel free to use our [sample apps](/extending-agent-app/agent-app-sdk/#sample-apps).
+
+As for the **widget placement**, you can choose one of the following locations: the Details section, Fullscreen apps, or Settings. To create a Ticket Details widget, you need to edit all three required fields: Widget source URL, Widget placement, and Opening state. If you want to create Full Screen or Settings widgets, only Widget source URL and Widget placement fields are required. [Read more](/extending-agent-app/#app-locations) about app placements.
+
+<div style={{maxWidth: '800px', marginBottom: '30px'}}>
+  <img src="/images/getting-started/app-guides/helpdesk-widgets/livechat-app-helpdesk-widget-creating.png" alt="HelpDesk Widgets" width="1144px" height="841px"/>
+</div>
+
+**Opening state** allows you to add components to your application and see a live preview of what the app will look like in the **Opening state preview**. 
+
+<div style={{maxWidth: '800px', marginBottom: '30px'}}>
```

Comment:
If possible, I'd try to show everything in one image, preferably the 3rd one. 

Since the dropdown covers the "Opening state preview" field, you can cut out the dropdown and Figma and place it more on the right or you can manipulate its position in the browser's developer tools. Feel free to reach out for help.

Reply:
done :) It looks much simpler with one pic